spec/javascripts/components/story/expanded_story/expanded_story_task_spec.js

```diff
@@ -32,32 +33,66 @@ describe('<ExpandedStoryTask />', () => {
     expect(wrapper.text()).toContain(I18n.t('story.tasks'));
   });
 
-  it('disables the add task button if text area is empty', () => {
-    const { input } = setup();
-    const { button } = setup();
+  describe('when component is enabled', () => {
+    it('displays a task form', () => {
+      const { wrapper } = setup();
 
-    input.simulate('change', { target: { value: '' } });
-    expect(button.prop('disabled')).toBe(true);
-  });
+      expect(wrapper.exists('.task-form')).toBe(true);
+    });
 
-  describe('onHandleSubmit', () => {
-    it('calls onSave with a task', () => {
-      const task = 'New Task';
-      const onSaveSpy = sinon.spy();
-      const { wrapperInstance } = setup({ onSave: onSaveSpy });
-      wrapperInstance.setState({ task })
+    it('disables the add task button if text area is empty', () => {
+      const { input } = setup();
```

Comment:
Do we need two setups? If no, we can just destructure both from the same call:

```suggestion
      const { input, button } = setup();
```

Reply:
I will check it.